### PR TITLE
Add Toil script for `adam convert`

### PR DESCRIPTION
**Pull request by heuermh**
Work towards #104

### CODE REVIEW

File: src/toil_scripts/adam_variants/convert_dbsnp.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -x -v
+
+python -m toil_scripts.adam_variants.convert_vcf \
+    aws:us-west-2:FIXME-toil-jobstore \
+    --num_nodes 3 \
```
Comment (heuermh) [inline src/toil_scripts/adam_variants/convert_dbsnp.sh]: Not sure how many nodes & what memory settings will work best for a file of this size

File: src/toil_scripts/adam_variants/convert_vcf.py [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+    Convert input VCF file into ADAM format.
+    """
+    log.write("adam convert\n")
+    log.flush()
+
+    adamFile = ".".join(os.path.splitext(inFile)[:-1])+".adam"
+    
+    call_adam(masterIP,
+              inputs,
+              ["vcf2adam",
```
Comment (heuermh) [inline src/toil_scripts/adam_variants/convert_vcf.py]: Do we need the ```--dict``` parameter?

File: src/toil_scripts/adam_variants/convert_vcf.py [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+    """
+    log.write("write data\n")
+    log.flush()
+
+    fileSystem, path = hdfsName.split('://')
+    nameOnly = path.split('/')[-1]
+    
+    uploadName = "%s/%s" % (inputs['outDir'], nameOnly)
+    if inputs['suffix']:
+        uploadName = uploadName.replace('.adam', '_%s.adam' % inputs['suffix'])
```
Comment (heuermh) [inline src/toil_scripts/adam_variants/convert_vcf.py]: ```'_%s.adam'``` &rarr; ```'%s.adam'```

**Comment by jvivian** [comment]
@heuermh — What is the status of this?

**Comment by heuermh** [comment]
Can close, #104 took care of this already

**Pull request closed without merging**